### ISSUE TITLE
Change map tile source

### ISSUE THREAD

**Issue by ianmcorvidae** [redfish64/TinyTravelTracker#33]
MapQuest has shut down the tile service which TTT is using: http://devblog.mapquest.com/2016/06/15/modernization-of-mapquest-results-in-changes-to-open-tile-access/

I'm not sure what the best alternative is, but new tiles will not download at this point, so some sort of change needs to be made. :(

**Comment by redfish64** [comment]
I'm working on it. I'm trying to integrate mapzen

**Comment by redfish64** [comment]
I finally finished converting to mapzen. It was quite a large change, so I had to remove some functionality, which I'll add back later (Specifically lines between points and selecting points to see what time there were created).

On the positive side, the maps look a lot better now, and mapzen has a lot of neat stuff such as directions and address finding I might be able integrate.

It should be in version 1.1.0.1 in f-droid (which should be available in a couple of days). It was quite a complex change and getting f-droid to build it was tricky, so we'll see if it works or not.

It also should be available on google play in a few hours as well.

**Comment by dreaming-augustin** [comment]
I couldn't let this pass without dropping by just to say: Thank you!
:)
Thank you for your hard work in maintaining this application.
Behind every cool open source software, there is a dedicated, generous and talented individual like you.

**Comment by redfish64** [comment]
Thanks! I appreciate it.

On Thu, Jul 28, 2016 at 9:26 AM, dreaming-augustin <notifications@github.com

**Comment by redfish64** [comment]
Sorry, I messed up the fdroid build. I just fixed it. 

I'm in China now, with crappy internet, so it's hard to get updates on these things.

**Comment by redfish64** [comment]
Finally made into fdroid as 1.1.5

**Comment by fdroiduser** [comment]
Hi! I just downloaded 1.1.7, and it's great! Thanks a lot for the work. I actually renders nicer than the previous version.

In addition, it also disabled lines and big points (which was a feature request of mine :-D)
So i'm very happy. If you add back the lines, please also add an option to keep them disabled ;)

Thanks once more for the work!

**Comment by redfish64** [comment]
Glad that it works. I think that vector graphics is the future, which is
why I went with it rather than trying to find an alternative source and
continue with the tile based system. Also, using an actively maintained
library for mapping is nice, because I can leverage their code.

Hopefully, I'll be able to add things like offline map download, navigation
and address finding, as well as different map skins. Take this with a grain
of salt, however, since after I finished the conversion, I took a long
break and haven't done much research into them, so I'm not sure whether
these things will be easy to add.
